contracts/extensions/votingReputation/VotingReputationStorage.sol

```diff
@@ -437,4 +440,85 @@ contract VotingReputationStorage is ColonyExtension, BasicMetaTransaction, Votin
 
     return expenditureAction;
   }
+
+  // Kept for backwards-compatibility with v9
+  function createClaimDelayAction(bytes memory action, uint256 value)
+    public
+    returns (bytes memory)
+  {
+    // See https://solidity.readthedocs.io/en/develop/abi-spec.html#use-of-dynamic-types
+    //  for documentation on how the action `bytes` is encoded
+    // In brief, the first byte32 is the length of the array. Then we have
+    //   4 bytes of function signature, following by an arbitrary number of
+    //   additional byte32 arguments. 32 in hex is 0x20, so every increment
+    //   of 0x20 represents advancing one byte, 4 is the function signature.
+    // So: 0x[length][sig][args...]
+
+    bytes4 sig = getSig(action);
+    assert(sig == SET_EXPENDITURE_STATE || sig == SET_EXPENDITURE_PAYOUT);
+
+    bytes4 functionSignature = SET_EXPENDITURE_STATE;
+
+    uint256 permissionDomainId;
+    uint256 childSkillIndex;
+    uint256 expenditureId;
+    uint256 storageSlot; // This value is only used if (sig == SET_EXPENDITURE_STATE)
+    uint256 expenditureSlot;
+
+    assembly {
+      permissionDomainId := mload(add(action, 0x24))
+      childSkillIndex := mload(add(action, 0x44))
+      expenditureId := mload(add(action, 0x64))
+      storageSlot := mload(add(action, 0x84))
+    }
+
+    // If we are editing the main expenditure struct
+    if (sig == SET_EXPENDITURE_STATE && storageSlot == 25) {
+      bytes memory mainClaimDelayAction = new bytes(4 + 32 * 11); // 356 bytes
```

Comment:
I just reintroduced the prior function as-was and kept the new version which I would want to maintain longer-term. I would rather keep them separate since `createClaimDelay` action is only for backwards-compatibility.